modules/EventEmitter.js

```diff
@@ -189,27 +195,52 @@ class EventEmitter {
                     ImportUtilities.unpackKeys(result.vertices, result.edges);
                 }
 
-                const dataimport =
-                    await Models.data_info.findOne({ where: { data_set_id: dataSetId } });
-
-                if (result.vertices.length === 0 || dataimport == null) {
+                if (result.vertices.length === 0) {
                     data.response.status(204);
                     data.response.send(result);
                 } else {
                     data.response.status(200);
-                    data.response.send({
-                        import: ImportUtilities.normalizeImport(
+                    if (!holdingData) {
+                        // Local import
+                        data.response.send({
+                            import: ImportUtilities.normalizeImport(
+                                dataSetId,
+                                result.vertices,
+                                result.edges,
+                            ),
+                            root_hash: dataInfo.root_hash,
+                            transaction: dataInfo.transaction_hash,
```

Comment:
why this variable (transaction) isn't called transaction_hash? 
it would make sense to call it transaction_hash